src/runtime/job.cpp

```diff
@@ -941,21 +979,10 @@ bool JobTable::wait(Runtime &runtime) {
           runtime.heap.guarantee(WJob::reserve());
           runtime.schedule(WJob::claim(runtime.heap, entry->job.get()));
           ++done;
-          if (!imp->batch && !entry->stderr_buf.empty()) {
-            if (entry->stderr_buf.back() != '\n') entry->stderr_buf.push_back('\n');
-            status_write(entry->job->stream_err.c_str(), entry->stderr_buf.data(),
-                         entry->stderr_buf.size());
-            entry->stderr_buf.clear();
-          }
```

Comment:
I'm not sure what this is doing, and it looks like it doesn't have a translation? Are we dropping/loosing a feature here?

Reply:
Nope :) I just cleaned it up a fuck load

This was line buffering. Line buffering is now handled by the StatusBuf directly